modules/110-istio/images/api-proxy/Dockerfile

```diff
@@ -1,11 +1,15 @@
-ARG BASE_ALPINE
+ARG BASE_ALT
 ARG BASE_GOLANG_16_ALPINE
 FROM $BASE_GOLANG_16_ALPINE as artifact
 WORKDIR /src/
 COPY main.go go.mod go.sum /src/
 RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags="-s -w" -o api-proxy main.go
 
-FROM $BASE_ALPINE
+FROM $BASE_ALT
```

Comment:
distroless pls.

Reply:
@apolovov the api-proxy, cni, operator migrated to Distroless.